allen-cell-animated/cell-feature-data
# Feature/cli single dataset

Problem
=======
What is the problem this work solves, including
a possible solution for #55

Solution
========
What I/we did to solve this problem
- added a script in `package.json` to enable single dataset validation 
- create a new file `check-single-dataset.js` to check for errors and validate a single dataset or a megaset  
- refactored `check-input-datasets.js` to incorporate `validateSingleDataset` within the loop
- removed the unused `validate.js`  

with @meganrm 

## Type of change
Please delete options that are not relevant.

* New feature (non-breaking change which adds functionality)
* This change requires a documentation update


Steps to Verify:
----------------
1. Run `npm run validate-single-dataset [RELATIVE-PATH-OF-SINGLEDATASET-OR-MEGASET]`


Keyfiles (delete if not relevant):
-----------------------
1. `check-input-datasets.js`
2. `check-single-dataset.js`
